documentation/jetty/modules/operations-guide/pages/deploy/index.adoc

```diff
@@ -427,24 +452,37 @@ $JETTY_BASE
 ├── environments
 │   ├── {ee-current}.xml
 │   └── {ee-current}-extra.xml
+├── start.d
+│   ├── ...
+│   └── {ee-current}-ext.ini
 └── webapps
     ├── wiki.properties
     └── wiki.xml
 ----
 
 [[jetty-web-xml]]
-== Jetty Context XML Inside `+*.war+` Files
+== In-Application Jetty Context XML File
 
-As discussed <<context-xml,above>>, the deployment of a web application can be specified with a Jetty Context XML file placed in the `$JETTY_HOME/webapps` directory.
+As discussed in <<context-xml,this section>>, the deployment of a web application can be specified with a Jetty Context XML file placed in the `$JETTY_HOME/webapps` directory.
 
-However, the web application can also be configured using a Jetty Context XML file placed _inside_ the `+*.war+` file or directory, under the `WEB-INF` directory.
-
-The files must be named either `WEB-INF/jetty-{ee-all}-web.xml` or `WEB-INF/jetty-web.xml`; the latter is only processed if the former(s) are missing.
+However, the deployment of a web application can also be configured using a Jetty Context XML file placed _inside_ the web application, both for <<web-application-format-core,Core web application deployments>> and <<web-application-format-jakarta,Jakarta web application deployments>>.
 
 This file can be useful:
 
 * To place all configuration inside your web application archive.
-* To perform configuration of the web application that can only occur _after_ the web application's class-path has been created, for example because it references classes present in `WEB-INF/classes` or `WEB-INF/lib`.
+* To perform configuration of the web application that can only occur _after_ the web application's class-path has been created, for example because it references classes present in web application itself.
```

Comment:
```suggestion
* To perform configuration of the web application that can only occur _after_ the application's class-path has been created, for example because it references classes present in the application itself.
```